tests/unit/service/test_infrastructure.py

```diff
@@ -8,9 +8,22 @@
 from ignition.service.messaging import Envelope, Message
 from ignition.service.logging import LM_HTTP_HEADER_PREFIX, LM_HTTP_HEADER_TXNID
 from ignition.service.messaging import Envelope, Message, TopicConfigProperties
+from ignition.utils.propvaluemap import PropValueMap
 
 class TestInfrastructureApiService(unittest.TestCase):
 
+    def __props_with_types(self, orig_props):
+        props = {}
+        for k, v in orig_props.items():
+            props[k] = {'type': 'string', 'value': v}
+        return props
+
+    def __propvaluemap(self, orig_props):
+        props = {}
+        for k, v in orig_props.items():
+            props[k] = {'type': 'string', 'value': v}
+        return PropValueMap(props)
```

Comment:
picky: couldn't you use a dictionary comprehension here to avoid the explicit construction of a variable?